search-api/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ $ENVIRONMENT == "PRODUCTION" ]]
+then
+	echo production 
+else
+	pip install ipython
+	pip install -r requirements.txt
+	chmod -R 777 /code/
```

Comment:
Why do we need these two lines: `chmod and chown code` ?

Reply:
Giving permission for making changes in files.

Reply:
Will remove it from production, only needed in development.

Reply:
For development, I think we will use docker-compose and we can just map the entire code volume and give it read-write permissions. This shouldn't be required. I am resolving this now, but if it is not required in future, we can remove it.